Packs/MailSenderNew/Integrations/MailSenderNew/MailSenderNew.py

```diff
@@ -386,12 +395,22 @@ def main():
                 cc = argToList(demisto.getArg('cc'))
                 bcc = argToList(demisto.getArg('bcc'))
                 str_msg = raw_message
+                html_body = raw_message
             else:
-                (str_msg, to, cc, bcc) = create_msg()
-
-            SERVER.sendmail(FROM, to + cc + bcc, str_msg)  # type: ignore[union-attr]
-            SERVER.quit()  # type: ignore[union-attr]
-            demisto.results('Mail sent successfully')
+                (_, html_body, str_msg, to, cc, bcc) = create_msg()
+
+            SERVER.sendmail(from_email, to + cc + bcc, str_msg)  # type: ignore[union-attr]
```

Comment:
General note: Strange that we don't distinguish between to, cc, and bcc

Reply:
@tkatzir we use the old python functions, can be upgraded in a future ticket to support cc,bcc

Reply:
Please create a dev-task, and leave this as is.

Reply:
ticket created